LukeMason33/CharityLocal
# Feature/about page

### Is this a fix or a feature?
- This is a feature that adds an About page to the website. This page is created in the file `About.js` that contains our `AboutPage` component. This component simply displays information about the app and about the author. There is one prop being passed to this component from our `App1 component which is the `charities` hook. The reason for this is so that if the user has searched for charities while on the site, they can go back to their previous search, but if they haven't searched yet, then no previous searched charities exist so they can only go back to the landing page.

### What is the change/fix?
- The change is the added `AboutPage` component within `About.js`. There is also the removal of our `Mission` directory because I'm not going to utilize it.

### Where should the reviewer start?
- The reviewer should start in `App.js` and examine the route to the About page. Then they should go to `About.js` and examine the new component `AboutPage` and notice how we are conditionally rendering the back to dashboard button based on the `charities` hook that's being passed to this component as a prop from `App` component.

### How should this be tested?
- This should be tested by opening up the app and ensuring that you can travel to the About page via the About button on the landing page. When on the About page, there should only be one button. Then go back to the landing page and do a search that will pull up charities. From the dashboard go to the About page via the About button in the header, where you should be brought to the About page and now there should be two nav buttons at the bottom of the screen.
